templates/overrides/infrastructure-vsphere/v0.6.4/infrastructure-components.yaml

```diff
@@ -0,0 +1,2912 @@
+apiVersion: v1
```

Comment:
@MnrGreg - did you have to change something to infrastructure-components.yaml ?

Reply:
no, just included in current dir for ease of use. will remove.